backend/app/models/field_type.rb

```diff
@@ -0,0 +1,104 @@
+# field_types table
+class FieldType < ActiveRecord::Base
+
+  # Create a field type
+  #
+  # @param field_type [Hash] the field type
+  # @option field_type[:name] [String] name
+  # @option field_type[:ftype] [String]  ftype
+  # @option field_type[:required] [String] required - interpreted as Boolen
+  # @option field_type[:array] [String] array - interpreted as Boolean
+  # @option field_type[:choices] [String] choices
+  # @option field_type[:allowable_field_types] [Array] array of allowable field types (for ftype = "sample")
+  # @return true
+  def self.create_sampletype(parent_id, field_type)
```

Comment:
since SampleType is another class having this be `create_sampletype` is confusing

Reply:
Agreed.  Let's discuss what to do here.  Maybe a name such as "create_parentclass_sampletype" (I generally don't like long names)

As background, there appear to be 2 "types" of field types, parent_class = OperationType and parent_class = SampleType.  Ideally they both get handled by the same "create" method, but that might be confusing since each one may have its own dependencies (I do not know yet what is involved when the parent_class =  OperationType).  That's why I pre-defined this method one to specifically handle only parent_class = SampleType.